### PR TITLE
Fixes #9918: Update maven compilation target to java8 in rudder-paren-pom

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9918

### CODE REVIEW

**Comment by amousset** [comment]
There is an unwanted commit

**Comment by peckpeck** [comment]
PR rebased

File: pom.xml [hunk 1]
```diff
@@ -148,7 +148,8 @@ limitations under the License.
         <configuration>
           <scalaCompatVersion>${scala-binary-version}</scalaCompatVersion>
           <args>
-            <arg>-target:jvm-1.7</arg>
+            <!-- require java8, cf http://www.rudder-project.org/redmine/issues/8963 -->
```
Comment (fanf) [inline pom.xml]: should be "cf http://www.rudder-project.org/redmine/issues/9917"

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**